hms-app/src/components/HackathonDetails.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import { getHackathonDetails } from '../actions/GetBackendData'
+import IdeaCardList from './IdeaCardList'
+
+type IProps = {
+  hackathonID: string
+}
+
+export default function HackathonDetails(props: IProps) {
+  const { hackathonID } = props
+  const [hackathonData, setHackathonData] = useState({
+    errorhackathonData: false,
+    isLoadinghackathonData: true,
+    title: 'string',
+    startDate: '',
+    endDate: '',
+    participantIds: [],
+    categoryIds: [],
+    ideas: [],
+  })
+
+  const {
+    errorhackathonData,
+    isLoadinghackathonData,
+    title,
+    startDate,
+    endDate,
+    participantIds,
+    categoryIds,
+    ideas,
+  } = hackathonData
+
+  const loadSelectedHackathon = () => {
+    getHackathonDetails(hackathonID.toString()).then(
```

Comment:
since you define it's type as `string` already` don't need to call `toString()`
```suggestion
    getHackathonDetails(hackathonID).then(
```